docs/package/gluon-ebtables-source-filter.rst

```diff
@@ -27,4 +27,6 @@ Example::
 
   prefix4 = '198.51.100.0/21',
   prefix6 = '2001:db8:8::/64',
-  extra_prefixes6 = { '2001:db8:9::/64', '2001:db8:100::/60' },
+  extra_prefixes6 = {
+    '2001:db8:9::/64', '2001:db8:100::/60'
```

Comment:
I think it would be nice to wrap this to have one prefix per line, and a comma after the last prefix for consistency.